source/draw/gpu/opengl/AndroidContext.ooc

```diff
@@ -141,5 +141,10 @@ AndroidContext: class extends OpenGLContext {
 		map add("startY", startY)
 		DrawState new(target) setMap(map) draw()
 	}
+	preallocate: override func (resolution: IntVector2D) { GraphicBufferYuv420Semiplanar free~all() }
+	preregister: override func (image: Image) {
+		if (image instanceOf(GraphicBufferYuv420Semiplanar))
+			(image as GraphicBufferYuv420Semiplanar) toRgba(this) referenceCount decrease()
```

Comment:
This is actually not a conversion, it's more a type cast. We simply interpret the YUV buffer as an RGBA buffer instead. It is then converted to YUV in the unpack/pack shaders.